chall02/darodrig.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+import sys
+
+def main():
+	chars = { 'A':'.-', 'B':'-...', 
+                    'C':'-.-.', 'D':'-..', 'E':'.', 
+                    'F':'..-.', 'G':'--.', 'H':'....', 
+                    'I':'..', 'J':'.---', 'K':'-.-', 
+                    'L':'.-..', 'M':'--', 'N':'-.', 
+                    'O':'---', 'P':'.--.', 'Q':'--.-', 
+                    'R':'.-.', 'S':'...', 'T':'-', 
+                    'U':'..-', 'V':'...-', 'W':'.--', 
+                    'X':'-..-', 'Y':'-.--', 'Z':'--..'}
+	if len(sys.argv) != 2 or sys.argv[1] == "":
+		print("usage: ./xlogin.py <a-zA-Z string>")
```

Comment:
Your file is named `darodrig.py` and could be named differently too, therefore your usage is incorrect.  
You could have made it dynamic this way:
```suggestion
		print("usage: %s <a-zA-Z string>" % sys.argv[0])
```